packages/insomnia-app/app/ui/components/modals/generate-code-modal.tsx

```diff
@@ -13,8 +14,8 @@ import ModalFooter from '../base/modal-footer';
 import ModalHeader from '../base/modal-header';
 import CodeEditor from '../codemirror/code-editor';
 
-const DEFAULT_TARGET = availableTargets().find(t => t.key === 'shell');
-const DEFAULT_CLIENT = DEFAULT_TARGET.clients.find(t => t.key === 'curl');
+const DEFAULT_TARGET = HTTPSnippet.availableTargets().find(t => t.key === 'shell');
+const DEFAULT_CLIENT = DEFAULT_TARGET?.clients.find(t => t.key === 'curl');
```

Comment:
right now, the type of this is `any`
```suggestion
const DEFAULT_CLIENT = DEFAULT_TARGET?.clients.find(t => t.key === 'curl') as HTTPSnippetClient;
```